contracts/LockedTokens.cdc

```diff
@@ -166,6 +173,7 @@ pub contract LockedTokens {
         /// Called by the admin every time a vesting release happens
         pub fun increaseUnlockLimit(delta: UFix64) {  
             self.unlockLimit = self.unlockLimit + delta
+            emit UnlockLimitIncreased(address: self.owner!.address, amount: delta)
```

Comment:
Is it worth adding a field to represent the new limit amount? This would make tracking the history of an account's limit easier as it can be established without keeping a running total.

I suggest this rather than changing the event to simply reflect the new level as the amount information is obviously also useful, and this naming reflects the function intention well.

Reply:
yeah I can add that field also